src/resolve/forceIgnore.ts

```diff
@@ -16,16 +17,26 @@ export class ForceIgnore {
 
   private readonly parser: Ignore;
   private readonly forceIgnoreDirectory: string;
-  private readonly contents?: string;
   private DEFAULT_IGNORE: string[] = ['**/*.dup', '**/.*', '**/package2-descriptor.json', '**/package2-manifest.json'];
 
   public constructor(forceIgnorePath = '') {
     try {
-      this.contents = readFileSync(forceIgnorePath, 'utf-8');
+      let contents = readFileSync(forceIgnorePath, 'utf-8');
       // check if file `.forceignore` exists
-      if (this.contents !== undefined) {
+      if (contents !== undefined) {
```

Comment:
Would `if (!contents)` be better here, or is this condition specifically for undefined?

Reply:
@jeffb-sfdc empty string is falsy--but if someone really makes an empty forceignore file we'll want honor that above our defaults

(plus that would be a behavior change and someone'll get bit by it 🤪 )